apps/src/templates/VerticalImageResourceCard.jsx

```diff
@@ -106,7 +106,7 @@ class VerticalImageResourceCard extends Component {
             href={link}
             color={Button.ButtonColor.gray}
             text={buttonText}
-            style={[styles.button]}
+            style={{...styles.button}}
```

Comment:
this was causing [tests to fail](https://drone.cdn-code.org/code-dot-org/code-dot-org/23271/1/3); i'm not completely sure why

Reply:
oh, that's a subtle (and mostly pointless, IMO) feature of radium. by default, the `style` attr only accepts an object, but any component using radium can pass an array instead (which just gets turned into an object)

Reply:
ahh, that makes sense then.